src/sentry_plugins/heroku/plugin.py

```diff
@@ -21,6 +22,28 @@ def handle(self, request):
             owner=user,
         )
 
+    def set_refs(self, release, **values):
+        # check if user exists, and then try to get refs based on version
+        if values.get('owner', None):
+            try:
+                repository = Repository.objects.get(
+                    organization_id=self.project.organization_id,
+                    name=ProjectOption.objects.get_value(
```

Comment:
i think we might want to check if we get a result back from `ProjectOption` before querying for repo